projects/crc-timing/testing/prepare_crc_timing.py

```
@@ -0,0 +1,211 @@
+    contents_result = remote_access.run_with_ansible_ssh_conf(
+        base_work_dir,
+        f"ls -1 '{crc_dir}' 2>/dev/null",
+        check=False,
+        capture_stdout=True
+    ).stdout.strip().split('\n')
+
+    for item in contents_result:
+        item_name = item.strip()
+        if not item_name: continue
+
+        item_path = crc_dir / item_path
```

Ruff (0.13.1)

Undefined name `item_path`

(F821)